production/kubernetes/agent-loki.yaml

```diff
@@ -59,6 +59,7 @@ spec:
       - args:
         - -config.file=/etc/agent/agent.yaml
         - -server.http.address=0.0.0.0:80
+        - -config.expand-env
```

Comment:
Thanks for the PR!

This file is generated from [Jsonnet](https://github.com/grafana/agent/blob/main/production/kubernetes/build/templates/loki/main.jsonnet) using the [v2 Tanka library](https://github.com/grafana/agent/tree/main/production/tanka/grafana-agent/v2). The Jsonnet would need to be updated and then regenerate this file (with `USE_CONTANIER=1 make generate-manifests`) before we can merge.

Reply:
Thank you for the guide!

I've updated the changes.